nuclear/tools/module.py

```diff
@@ -154,22 +190,45 @@ def generate_readme(parts):
 
         ## Dependencies
 
-        """)
+        """
+    )
+
+    return template.format(
+        className=parts[-1], classNameTitle=len(parts[-1]) * '=', closeNamespace='\n'.join(['}' for x in parts[:-1]])
+    )
 
-    return template.format(className=parts[-1]
-                         , classNameTitle = len(parts[-1]) * '='
-                         , closeNamespace = '\n'.join(['}' for x in parts[:-1]]))
 
 def generate_test(parts):
-    template = textwrap.dedent("""\
+    template = textwrap.dedent(
+        """\
         // Uncomment this line when other test files are added
         //#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
         //#include <catch.hpp>
 
         // Remove this line when test files are added
         int main() {{ return 0; }}
-        """)
+        """
+    )
 
     return template.format()
 
 
+def generate_python(parts):
+    template = textwrap.dedent(
+        """\
+        #!/usr/bin/env python3
+
+        from nuclear import Reactor, on, Trigger, Single, With, Every
+
+        @Reactor
+        class {className}(object):
+            def __init__(self):
+                # Constructor for {className}
+
+            @on(Configuration('{className}.yaml'))
+            def {className}_configfuration(self, config):
```

Comment:
Did you actually get this code to work?

Reply:
No, I should probably comment this out here for now. We need to create more bindings in general though (extensions and utilities)